tests/unit/test_tenant.py

```diff
@@ -10,6 +12,28 @@ def setup(self):
         self.custom_404_msg = "Tenant not found"
         self.schema = TenantSchema
 
+    def test_only_active_lease_returned(self, faker, create_lease, create_tenant):
+        tenant = create_tenant()
+
+        # Create an expired lease for the tenant
+        lease_expired_end = faker.date_time_this_decade(
+            before_now=True, after_now=False
+        )
+        lease_expired_start = lease_expired_end - datetime.timedelta(days=365)
+        lease_expired = create_lease(
+            tenant=tenant,
+            dateTimeStart=lease_expired_start,
+            dateTimeEnd=lease_expired_end,
+        )
+
+        # Create an active lease for the tenant
+        lease_active = create_lease(
+            tenant=tenant, property=PropertyModel.find_by_id(lease_expired.propertyID)
```

Comment:
FYI: We shouldn't be using the `find_by_id` method. I wonder if there is a way to put a deprecation warning for whenever a developer users that method :-) All new code should be using the `find` method as I would like to remove the `find_by_id` method some day.

Reply:
Done!

Reply:
Done!